test/jdk/sun/security/jca/app-security.properties

```diff
@@ -0,0 +1 @@
+jdk.security.provider.preferred=KeyStore.PKCS12:JipherJCE
```

Comment:
How about just `NonExistingProvider`? The `JipherJCE` name does not make it clear what you intend it to do.